e2e/features/step_definitions/reissue-document.steps.ts

```diff
@@ -0,0 +1,30 @@
+import { Then, When } from 'cucumber';
+import { AnyCcdPage } from '../../pages/any-ccd.page';
+import { expect } from 'chai';
+import { browser } from 'protractor';
+const anyCcdPage = new AnyCcdPage();
+
+When('resend only to appellant and not to representative', async function () {
+    await browser.sleep(2000);
+    await anyCcdPage.clickElementById('resendToAppellant_Yes');
+    await anyCcdPage.clickElementById('resendToRepresentative_No');
+    await anyCcdPage.click('Continue');
+});
+
+When('I submit “Reissue document”', async function () {
```

Comment:
We don't need this method again as we already have it here https://github.com/hmcts/sscs-ccd-e2e-tests/blob/master/e2e/features/step_definitions/response-reviewed.steps.ts#L14